contrib/upgrade.sh

```diff
@@ -0,0 +1,83 @@
+#!/usr/bin/env bash
+set -e
+
+# This is an update script for gitea installed via the binary distribution
+# from dl.gitea.io on linux as systemd service. It performs a backup and updates
+# Gitea in place.
+# Depends on: bash, curl, xz, sha256sum, gpg, which. optionally jq.
+# Usage:      [environment vars] upgrade.sh [version]
+#   See section below for available environment vars.
+#   When no version is specified, updates to the latest release.
+# Examples:
+#   upgrade.sh 1.15.10
+#   giteahome=/opt/gitea giteaconf=$giteahome/app.ini upgrade.sh
+
+# apply variables from environment
+: ${giteabin:=/usr/local/bin/gitea}
+: ${giteahome:=/var/lib/gitea}
+: ${giteaconf:=/etc/gitea/app.ini}
+: ${giteauser:=git}
+: ${sudocmd:=sudo}
+: ${arch:=linux-amd64}
+: ${backupopts:=--skip-lfs-data --skip-repository --skip-attachment-data --skip-log}
```

Comment:
What exactly is the reason for excluding these things from the backup by default?

I mean, why would that be wanted? To save some space and potentially lose all that data?
Or will those data not be changed by the upgrade and hence can be ignored?

Reply:
The idea was to speed it up to save downtime & storage space. But yeah it's not a good default, I'll revert this.